ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -913,11 +914,21 @@ export class FrontstageDef {
     const widgetDef = this.findWidgetDef(tabId);
     if (!widgetDef) return;
 
-    const adjustmentWidth = ProcessDetector.isElectronAppFrontend ? 16 : 0;
-    const adjustmentHeight = ProcessDetector.isElectronAppFrontend ? 39 : 0;
+    let width =
+      childWindow.currentBrowser === "chromium based edge"
+        ? childWindow.outerWidth
+        : childWindow.innerWidth;
+    let height =
+      childWindow.currentBrowser === "chromium based edge"
+        ? childWindow.outerHeight
+        : childWindow.innerHeight;
+    if (childWindow.deltaHeight && childWindow.deltaWidth) {
```

Comment:
Could their be only a delta in either ? (deltaHeight === 2, deltaWidth === 0)

Reply:
There were always be a delta width and delta height, even if it's just 0. All `ChildWindow` props are optional parameters because they aren't set when the window is initially created and there were errors about that. So I don't think changing it is necessary

Reply:
With the current `if` condition, if either is `0` none will be applied. (`0` is falsy, and the `&&` makes it so both need to be truthy) Should be a `||`